examples/bridge-app/linux/main.cpp

```diff
@@ -1024,6 +1024,7 @@ int main(int argc, char * argv[])
 
     // Run CHIP
 
+    ChipLinuxAppInitNetworkCommissioning();
     ApplicationInit();
     registerAttributeAccessOverride(&gPowerAttrAccess);
     chip::DeviceLayer::PlatformMgr().RunEventLoop();
```

Comment:
So here is the question: why is bridge-app not calling `ChipLinuxAppMainLoop`?  It seems like it should!

It looks like whoever wrote this code basically didn't undertand the startup sequence.  What this code should do is:

```
ChipLinuxAppMainLoop();
```

and that's it.  A bunch of the things it does that ChipLinuxAppInit handles can be removed.  The setup of all the bridge endpoints and the `registerAttributeAccessOverride(&gPowerAttrAccess);` call should move into the now-empty `ApplicationInit()` function in this very file.  `ChipLinuxAppMainLoop()` will then call that `ApplicationInit()` at the right spot in startup, set up the network commissioning stuff, print the onboarding codes (something else the bridge app is missing), etc.

Reply:
@bzbarsky-apple , Thanks for your advice! I misunderstood that the person who wrote this code intended to initialize differently than the `ChipLinuxAppMainLoop` function. I will fix it.